Dockerfile

```
@@ -0,0 +1,12 @@
+FROM node:alpine AS stage
+
+WORKDIR /extension
+
+COPY . .
+RUN npm install -g pnpm
+RUN pnpm install
+RUN pnpm build
+
+FROM node:alpine
```

you can use the smallest alpine image
```suggestion
FROM alpine:3.20
```